crates/shared/src/arguments.rs

```diff
@@ -427,6 +435,12 @@ impl Display for Arguments {
             self.balancer_pool_deny_list
         )?;
         display_secret_option(f, "solver_competition_auth", &self.solver_competition_auth)?;
+        display_option(
+            f,
+            "settlement_contract_address",
+            &self.settlement_contract_address,
+        )?;
+        display_option(f, "native_token_address", &self.native_token_address)?;
```

Comment:
Didn't `H160` need to use `Debug` formatting to get the complete readable address?

Reply:
Nice catch!

Reply:
fixed